Modules/IxNetRestApiPortMgmt.py

```diff
@@ -251,8 +202,8 @@ def getAllVportList(self):
         Returns
             List of vports: ['/api/v1/sessions/1/ixnetwork/vport/1', '/api/v1/sessions/1/ixnetwork/vport/2']
         """
-        response = self.ixnObj.get(self.ixnObj.sessionUrl+'/vport')
-        vportList = ['%s' % vport['links'][0]['href'] for vport in response.json()]
+        self.ixnObj.logInfo("Get vports for all ports")
+        vportList = [vport for vport in self.ixNetObj.Vport.find()]
```

Comment:
Here list comprehensive not required, use direct "find" method to return list of vports.